helm/prefect-server/README.md

```diff
@@ -1,31 +1,132 @@
-**EXPERIMENTAL: This chart guarantees no compatability with future versions and is subject to change**
-
 # Prefect Server Helm Chart
 
+<!-- TOC -->
+
+- [Usage](#usage)
+    - [Installing released versions](#installing-released-versions)
+        - [Verifying package integrity](#verifying-package-integrity)
+    - [Installing development versions](#installing-development-versions)
+    - [Upgrading](#upgrading)
+        - [Important notes about upgrading](#important-notes-about-upgrading)
+- [Options:](#options)
+    - [Tenant](#tenant)
+    - [KubernetesAgent](#kubernetesagent)
+    - [Database](#database)
+- [Versioning](#versioning)
+    - [Development](#development)
+- [Connecting to your Server](#connecting-to-your-server)
+    - [Configure `prefect` to point to your service](#configure-prefect-to-point-to-your-service)
+    - [Forward your service to localhost](#forward-your-service-to-localhost)
+- [Troubleshooting](#troubleshooting)
+    - [The UI loads but the dashboard is blank](#the-ui-loads-but-the-dashboard-is-blank)
+    - [The UI loads but cannot connect to the API](#the-ui-loads-but-cannot-connect-to-the-api)
+    - [The database deploys correctly but other services fail with "bad password"](#the-database-deploys-correctly-but-other-services-fail-with-bad-password)
+    - [The agent is running but when I look at the logs I am getting GraphQL errors](#the-agent-is-running-but-when-i-look-at-the-logs-i-am-getting-graphql-errors)
+    - [Upgrading Hasura](#upgrading-hasura)
+
+<!-- /TOC -->
+
 ## Usage
 
+### Installing released versions
+
+The Helm chart is automatically versioned and released alongside Server. 
+For each Github Release of Server there is a corresponding version of the Helm chart.
+The charts are hosted in a [Helm repository](https://helm.sh/docs/chart_repository/) deployed to the web courtesy of Github Pages.
+
+1. Let you local Helm manager know about the repository.
+
+    ```
+    $ helm repo add prefecthq https://prefecthq.github.io/server/
+    ```
+
+2. Sync versions available in the repo to your local cache.
+
+    ```
+    $ helm repo update
+    ```
+
+3. Search for available charts and versions
+
+    ```
+    $ helm search repo [name]
+    ```
+
+4. Install the Helm chart
+
+    ```
+    $ helm install prefecthq/prefect-server --generate-name
```

Comment:
I might add a few more common options to the example install command (including the release name, values file, and namespace). You might also encourage the use of `helm upgrade --install` instead of `helm install`, as the former can be used for both installs and upgrades. 

See e.g. https://gateway.dask.org/install-kube.html#install-the-helm-chart for what I'd expect:

```
helm upgrade --install \
    --namespace $NAMESPACE \
    --version $VERSION \
    --values path/to/your/config.yaml \
    $RELEASE \
    prefecthq/prefect-server
```

Reply:
Thanks for taking a look. Those docs look a lot more approachable than this page but I don't think I'll go much further for now. I've taken a shot at this request though and I agree it helps.